documentation/release_notes.rst

```diff
@@ -8,6 +8,81 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.8.0
+===============
+
+New Features
+------------
+- Added support of host policies for ``histogram`` algorithms.
+- Added support for specification of a size limit for the output range in the range-based ``merge`` algorithm.
+- Improved performance of the ``merge`` and ``merge_sort`` algorithms with device policies for large data sizes. 
+- Improved performance of ``set_intersection``, ``set_union``, ``set_difference``, ``set_symmetric_difference``
+  algorithms with device policies.
+- Improved performance of ``reduce_by_segment`` with device policies for GPUs with sub-group sizes of 32.
+- Improved performance of ``adjacent_difference``, ``binary_search``, ``copy``, ``copy_n``, ``fill``, ``fill_n``,
+  ``for_each``, ``for_each_n``, ``generate``, ``generate_n``, ``lower_bound``, ``move``, ``replace``, ``replace_copy``,
+  ``replace_copy_if``, ``replace_if``, ``reverse``, ``reverse_copy``, ``rotate``, ``rotate_copy``, ``shift_left``,
+  ``shift_right``, ``swap_ranges``, ``transform``, ``transform_if``, and ``upper_bound`` with device policies along with
+  the analogous uninitialized memory operations.
+- Updated oneDPL use of SYCL to ensure the library is SYCL 2020 compliant.
+
+Fixed Issues
+------------
+- Fixed an issue with ``drop_view`` in the experimental range-based API.
+- Fixed an issue with range-based ``sort`` and ``stable_sort`` algorithms on Windows
```

Comment:
Can this note be more specific, e.g. hint what the issue was? @MikeDvorskiy, could you help with that?